src/NHSD.BuyingCatalogue.Identity.Common/Email/MimeKitExtensions.cs

```diff
@@ -27,7 +28,7 @@ internal static MimeMessage AsMimeMessage(this EmailMessage emailMessage)
 
             var message = new MimeMessage
             {
-                Subject = emailMessage.Subject ?? string.Empty,
+                Subject = $"{emailSubjectPrefix} {emailMessage.Subject}" ?? string.Empty,
```

Comment:
If the prefix is an empty string we probably do want to trim the whole subject line e.g. Subject = " Test email". Applying a trim will remove the leading space e.g. "Test email".

Reply:
👍